apps/meteor/client/providers/AuthenticationProvider/AuthenticationProvider.tsx

```diff
@@ -114,11 +114,7 @@ const AuthenticationProvider = ({ children }: AuthenticationProviderProps): Reac
 					Accounts._unstoreLoginToken = _unstoreLoginToken;
 				};
 			},
-
-			queryLoginServices: {
-				getCurrentValue: () => loginServices.getLoginServiceButtons(),
-				subscribe: (onStoreChange: () => void) => loginServices.on('changed', onStoreChange),
-			},
+			queryLoginServices: () => [(cb) => loginServices.on('changed', cb), () => loginServices.getLoginServiceButtons()],
```

Comment:
not following why changing this structure is required

Reply:
besides trying to have a pattern when using such thing, since its not a function, when using it on the mock context it throws an infinity loop error